app/controllers/board_sections_controller.rb

```diff
@@ -86,6 +87,18 @@ def require_permission
     end
   end
 
+  def og_data
+    desc = []
+    desc << @board_section.board.writers.pluck(:username).sort_by(&:downcase).join(', ') unless @board_section.board.open_to_anyone?
+    post_count = @board_section.posts.count
+    desc << "#{post_count} " + "post".pluralize(post_count)
```

Comment:
Add code for descriptions if present, since there's a migration in for that now.